yo1k/tic_tac_toe/core/game.py

```diff
@@ -179,13 +182,13 @@ def next(self) -> Optional[Action]:
 class Logic:
     """Game logic."""
 
-    def __init__(self, action_queues: Sequence[ActionQueue]):
+    def __init__(self, action_queues: MutableSequence[ActionQueue]):
```

Comment:
Why does this parameter need to be `MutableSequence`? `Logic` is not supposed to either add or remove any elements from it.

Reply:
Done.